### ISSUE TITLE
Example to show custom logic in evaluation step (could be in between training steps) based on logits

### ISSUE THREAD

**Issue by snowmanwwg** [NVIDIA-NeMo/Megatron-Bridge#336]
Show an example that users can compute custom logic based on logits exposed in OOTB NeMo/Megatron-bridge.

Related discussions - https://nvidia.slack.com/archives/C083Y1XED43/p1751910081883179

**Comment by yaoyu-33** [comment]
Closing as part of issue cleanup — this issue has been stale with no recent activity. If you still need help, please open a new issue with updated details. Thank you!
